content/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -66,9 +152,9 @@ Examples for how to use this function can be found in the
 
 You don't need to do anything. Full Kubeflow deployment has already deployed the `user-gcp-sa` secret for you.
 
-#### Cluster setup to use use_gcp_secret for Pipelines Standalone and Hosted GCP ML Pipelines
+#### Cluster setup to use use_gcp_secret for Pipelines Standalone and AI Platform Pipelines
 
-Pipelines Standalone and Hosted GCP ML Pipelines require you to manually set up the `user-gcp-sa` secret used by `use_gcp_secret`. 
+Pipelines Standalone and AI Platform Pipelines require your manual setup for the `user-gcp-sa` secret used by `use_gcp_secret`.
```

Comment:
please check my doc sent yesterday on removing it from CloudSQL and Minio. So when we support WI or GCE-DefaultAccount, no need to require user to setup this. Unless user wants to use it in user-made pipelines. Aka, it will be optional by user and no need to address here.

Reply:
I totally agree.

Note that this section is titled "Cluster setup to use use_gcp_secret for Pipelines Standalone and AI Platform Pipelines". When users do not use it, they can just skip this section.